internal/dkg/state_machine.go

```diff
@@ -438,6 +436,9 @@ func (d *DBState) Accepted(me *drand.Participant) (*DBState, error) {
 		return nil, ErrCannotAcceptProposalWhereJoining
 	}
 
+	d.Acceptors = append(d.Acceptors, me)
+	d.Rejectors = util.Without(d.Rejectors, me)
```

Comment:
Is this just to have to correct output in `dkg status` ?
If so, don't you need to do this for `func (d *DBState) Rejected(me *drand.Participant)` as well?

Reply:
good point, will add this!